storm-core/src/jvm/org/apache/storm/testing/staticmocking/MockedConfigUtils.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version
+ * 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.storm.testing.staticmocking;
+
+import org.apache.storm.utils.ConfigUtils;
+
+public class MockedConfigUtils extends ConfigUtils implements AutoCloseable {
```

Comment:
> Derek's repository has a file package-info.java, which shows how to use it. Do I need to include that within this pull request or wait util JIRA to do that? @d2r @revans2

For what it's worth, it is fine with me if you want to include that file in this pull request.